src/EA.Weee.RequestHandlers.Tests.Unit/AatfEvidence/GetAatfNotesRequestHandlerTests.cs

```diff
@@ -3,9 +3,11 @@
     using AutoFixture;
     using Core.Helpers;
```

Comment:
Need to update the HandleAsync_GivenRequest_EvidenceDataAccessShouldBeCalledOnce to expand on the following to include the compliance year.

A.CallTo(() => noteDataAccess.GetAllNotes(A<NoteFilter>.That.Matches(e => 
                e.OrganisationId.Equals(request.OrganisationId) && 
                e.AatfId.Equals(request.AatfId) && 
                e.AllowedStatuses.SequenceEqual(status) &&
                e.SchemeId == request.RecipientId))).MustHaveHappenedOnceExactly();